sound/soc/sof/loader.c

```diff
@@ -20,7 +20,7 @@ static int get_ext_windows(struct snd_sof_dev *sdev,
 {
 	const struct sof_ipc_window *w =
 		container_of(ext_hdr, struct sof_ipc_window, ext_hdr);
-	size_t w_size = struct_size(w, window, w->num_windows);
+	size_t w_size = sizeof(struct sof_ipc_window);
```

Comment:
@ktrzcinx this wont be backward compatible with abi minor 16 right?

Reply:
Good point, I should keep old size calculation to keep backward compatibility.